<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConverterDialog</class>
 <widget class="QDialog" name="ConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>224</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelSource">
       <property name="text">
        <string>&amp;Source file:</string>
       </property>
       <property name="buddy">
        <cstring>editSource</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editSource"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="buttonBrowseSource">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSizes">
       <property name="text">
        <string>Si&amp;zes:</string>
       </property>
       <property name="buddy">
        <cstring>listWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QListWidget" name="listWidget">
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>16 px</string>
        </property>
        <property name="checkState">
         <enum>Checked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32 px</string>
        </property>
        <property name="checkState">
         <enum>Checked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48 px</string>
        </property>
        <property name="checkState">
         <enum>Checked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64 px</string>
        </property>
        <property name="checkState">
         <enum>Checked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128 px</string>
        </property>
        <property name="checkState">
         <enum>Checked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256 px</string>
        </property>
        <property name="checkState">
         <enum>Checked</enum>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDestination">
       <property name="text">
        <string>Save &amp;to:</string>
       </property>
       <property name="buddy">
        <cstring>editDestination</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editDestination"/>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="buttonBrowseDestination">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editSource</tabstop>
  <tabstop>buttonBrowseSource</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>editDestination</tabstop>
  <tabstop>buttonBrowseDestination</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConverterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConverterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
